free-programming-books-id.md

```diff
@@ -103,12 +104,13 @@
 
 * [Belajar Python](http://www.belajarpython.com)
 * [Dasar Pemrograman Python](https://www.pythonindo.com/tutorial-python-dasar/)
+* [Dasar Pengenalan Pemrograman Python](https://play.google.com/store/books/details/Rolly_Maulana_Awangga_Dasar_dasar_Python?id=YpzDDwAAQBAJ) - Rolly Maulana AwanggaRayhan, Google Play Books (account required)
 * [Database PostgreSQL, Pemrograman Python, dan SMS Gateway](http://rab.co.id/files/python/bukupython2.pdf.gz) - (PDF)
 * [Kursus Singkat Machine Learning dengan TensorFlow API](https://developers.google.com/machine-learning/crash-course?hl=id)
 * [Tutorial Python](https://docs.python.org/id/3.8/tutorial/)
 * [Workshop Python 101](http://sakti.github.io/python101/)
 
 
 ### Vue
-
+* [Belajar Vue.js](https://variancode.com/belajar-vue-js/)
```

Comment:
Solve lint errors + add book author as seen in the footer of webpage

```suggestion

* [Belajar Vue.js](https://variancode.com/belajar-vue-js/) - Varian Hermianto
```

Reply:
Resolved in commit: https://github.com/EbookFoundation/free-programming-books/pull/6163/commits/6d14c3491378e6aada0c364d3d854c92bfa3341f